packages/cubejs-druid-driver/src/DruidClient.ts

```diff
@@ -64,7 +64,7 @@ export class DruidClient {
         });
 
         if (cancelled) {
-          await this.cancel(response.headers['x-druid-sql-query-id']);
+          await this.cancel(response.headers['x-druid-sql-query-id'] || '');
```

Comment:
If there is no `x-druid-sql-query-id,`, we don't need to call the `cancel` method, + we need to log an error.